tests/test_select.py

```diff
@@ -147,16 +149,19 @@ def test_select_left_join_with_extra_space_in_middle():
     assert_table_lineage_equal("SELECT * FROM tab1 LEFT  JOIN tab2", {"tab1", "tab2"})
 
 
+# deactivated for sqlfluff since it can not be parsed properly
 def test_select_left_semi_join():
     assert_table_lineage_equal(
-        "SELECT * FROM tab1 LEFT SEMI JOIN tab2", {"tab1", "tab2"}
+        "SELECT * FROM tab1 LEFT SEMI JOIN tab2", {"tab1", "tab2"}, test_sqlfluff=False
```

Comment:
Looks like we can make this work by specifying dialect="hive", am I missing anything here?